app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryController.kt

```diff
@@ -240,6 +245,8 @@ class LibraryController(
     var staggeredBundle: Parcelable? = null
     private var staggeredObserver: ViewTreeObserver.OnGlobalLayoutListener? = null
 
+    // Dynamically injected into the search bar, controls category visibility during search
+    private lateinit var showAllCategoriesView: ImageView
```

Comment:
I'd rather this just be a nullable ImageView instead of lateinit, so we can not have this crash when something unexpected happens

Reply:
Resolved by https://github.com/Jays2Kings/tachiyomiJ2K/pull/1386/commits/f70ec39a808ce33a6f05364b287b0d204d53806b